server/testFixture/one/imports.omt

```diff
@@ -1,17 +1,15 @@
 import:
-    module:Declared:
-    -   DeclaredQuery
-     ../relative.omt:
+    ../relative.omt:
     -   RelativeAction
-    @one/shorthanded.omt:
+    '@one/shorthanded.omt':
```

Comment:
The whole point of this line is to check whether the plugin can handle the unquoted version. If that is no longer something the plugin should be able to handle, remove this entry.

Reply:
When having imports unquoted, it is not valid YAML anymore. I will remove this entry.